release_notes.md

```diff
@@ -6,10 +6,33 @@ permalink: /release_notes/
 
 ## [Unreleased](https://github.com/triplea-game/triplea/releases)
 
-See the [full list here](https://github.com/triplea-game/triplea/pulls?q=merged%3A%3E%3D2017-11-02T18%3A11%3A00-04%3A00).
+See the [full list here](https://github.com/triplea-game/triplea/pulls?q=merged%3A%3E%3D2017-11-14T00%3A21%3A00-05%3A00).
+
+## [1.9.0.0.7527](https://github.com/triplea-game/triplea/releases/tag/1.9.0.0.7527) - November 14th 2017
+
+### Added
+* Wait indicator while loading default map at startup (#2573)
+* Wait indicator while parsing maps (#2572)
```

Comment:
Probably worth noting that maps are now parsed in parallel when loading the available games due to triplea-game/triplea#2572

Reply:
Done in rebased 2d5ca79.  I combined it with the the "Parse bot maps in parallel" item under the Changed section.